src/modules/awake/Awake/Core/InternalConstants.cs

```diff
@@ -8,5 +8,6 @@ internal static class InternalConstants
     {
         internal const string AppName = "Awake";
         internal const string FullAppName = "PowerToys " + AppName;
+        internal const string TrayWindowId = "WindowsForms10.Window.0.app.0.";
```

Comment:
Does this need to be specifically this? Is there a way to make it clearer this comes from PowerToys Awake?

Reply:
Unfortunately no - the window ID is generated by Windows when the tray icon is generated. On the bright side, when I am looking for windows that need to be handled, I am only snapping to PowerToys Awake as the process where that is applicable, so it should never really try and do anything funky with unrelated windows.